src/runtime/virtcontainers/clh.go

```diff
@@ -243,15 +243,15 @@ func (s *CloudHypervisorState) reset() {
 }
 
 type cloudHypervisor struct {
+	vmconfig        chclient.VmConfig
 	console         console.Console
 	virtiofsDaemon  VirtiofsDaemon
-	APIClient       clhClient
 	ctx             context.Context
-	id              string
+	APIClient       clhClient
```

Comment:
At first sight, this looks totally unrelated to the description of the commit, and it's unclear to me why you would even need the change.

Reply:
May be due to rebase, will check.

Reply:
This still looks wrong in this commit, since you introduce the change in this series in de39fb7d38, only to revert it here. But OK, I understand this went through several rebases, and I won't block for this. It's just a bit of noise in the history.